src/badge/index.tsx

```diff
@@ -0,0 +1,26 @@
+import React from 'react'
+import cc from 'classcat'
+import isEmpty from 'lodash.isempty'
+
+import style from 'badge/style'
+
+interface BadgeProps {
+  readonly className?: Classcat.Class,
+  readonly children: string | JSX.Element,
+  readonly ariaLabel?: string,
+}
+
+const Badge = ({ className, children, ariaLabel }: BadgeProps) => {
+  if (isEmpty(children) || children === '0') {
```

Comment:
Can't it be a number?

Reply:
Strictly speaking, yes, but I'd rather have a string if one wants to use it for stuff like "NEW".

Reply:
okay, but in the tests you are actually giving a number so it should be managed by Typescript as well I guess?

Reply:
Looks like Typescript is set to not agree with `number` for `aria-label`
<img width="516" alt="screen shot 2018-09-26 at 15 06 12" src="https://user-images.githubusercontent.com/373381/46081882-f1780100-c19d-11e8-9d76-891b52c3b97b.png">

Reply:
I was talking about the children for the number

Reply:
Hum, I'm not sure I'm actually giving it a number in the test, `12` here should be rendered as a string `<Badge>12</Badge>`

Reply:
Ok as discussed with @wakooka I'm simply gonna remove the `children === "0"` since this logic should not be handled in `ui-library`